server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/webadmin/ImapRoutes.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.james.imapserver.webadmin;
```

Comment:
You are missing the compulsary Apache V2 license file header

Reply:
Licence header added

Reply:
@chibenwa btw licencing headers - it seems not all files have them (I just saw https://github.com/apache/james-project/blob/master/server/protocols/protocols-library/src/test/java/org/apache/james/protocols/lib/mock/MockProtocolHandlerLoader.java)...  Maybe use https://github.com/mathieucarbou/license-maven-plugin for checking and formatting?